data-modeling/delegate-types/README.md

```diff
@@ -0,0 +1,68 @@
+# Data modeling: Delegate types
+
+This example shows you how to model [delegate types](https://edgeapi.rubyonrails.org/classes/ActiveRecord/DelegatedType.html) using Prisma, and use [Prisma Client](https://www.prisma.io/client) in a **TypeScript script** to read and write data.
```

Comment:
Since the language is not too relevant for the data modeling example I'd de-emphasize it at this point

```suggestion
This example shows you how to model [delegate types](https://edgeapi.rubyonrails.org/classes/ActiveRecord/DelegatedType.html) using Prisma, and use [Prisma Client](https://www.prisma.io/client) in a TypeScript script to read and write data.
```